com.unity.shadergraph/Editor/Util/AssertHelpers.cs

```diff
@@ -1,20 +1,20 @@
-using NUnit.Framework;
+using UnityEngine.Assertions;
 
 namespace UnityEditor.ShaderGraph
 {
     static class AssertHelpers
     {
-        public static void IsNotNull(object anObject, string message, params object[] args)
```

Comment:
It seems no one was using the args, but you could keep this in as it can be handy (looks to be for string formatting). It looks like NUnit defines this as:
```
static public void Fail(string? message, params object?[]? args)
        {
            if (message == null) message = string.Empty;
            else if (args != null && args.Length > 0)
                message = string.Format(message, args);

            ReportFailure(message);
        }
```

If you want to leave it alone that's fine too as string interpolation is an easy enough replacement

Reply:
I wasn't sure if that would cause problems (function signature mismatch) as our internal stuff doesn't define them the same way NUnit does.  I don't feel strongly one way or the other, but none of the actual calls to these actually provided that final argument, so I dropped it.

Reply:
We could keep the params arg on our Fail call, but the Unity version of IsNotNull doesn't offer it so we'd have to implement it ourselves. I think consistency of not including it is fine; we can always add it later if absolutely need it.